components/overlay/reports/1-a11y.json

```
@@ -0,0 +1,1416 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/wet-boew/demos/overlay/overlay-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "The main content of the page which contains the overlay plugin HTML code and its transformation",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "main"
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Hani Ahmed (Github: @heansih)",
+      "foaf:homepage": "https://github.com/heansih",
+      "@type": ["earl:Assertor", "foaf:Person"],
+      "foaf:member": {
+        "@id": "_:wave_browser_extension",
+        "doap:name": "WAVE web accessibility evaluation tool, browser extension",
+        "doap:homepage": "https://wave.webaim.org/extension/",
+        "@type": [ "earl:Assertor", "earl:Software" ]
+      }
+    }
```

This should be updated with your name and github handle. It do identify who created this assessment report

```suggestion
      "foaf:name": "Your Name (Github: @yourGitHubHandle)",
      "foaf:homepage": "https://github.com/yourGitHubHandle",
      "@type": ["earl:Assertor", "foaf:Person"]
    }
```